src/components/Image/Image.md

```diff
@@ -160,13 +158,15 @@ Handling load state:
 const { ImageFit, ImageLoadState } = require('.');
 
 const badImage = '404.png';
-const yammerLogoOrange = 'yammer-logo-orange.jpg';
+const yammerLogo = 'yammer-logo.png';
```

Comment:
this should be yammerLogoOrange, not yammerLogo, as it's showing a different fallback image when the image fails to load

Reply:
Changed. The orange logo was ugly af, and also a JPEG. I asked Design for new assets.

Reply:
lol